nltk/embedding/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Natural Language Toolkit: word embedding package
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+# Import Gensim if installed
+try:
```

Comment:
To make installations easier, it might be useful to define extra dependencies, see https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies

something like
```python
setup(
    ...
    extras_require = {
        'everything':  ['gensim'],
    }
)
```

then if one wants to install NLTK with all it's dependencies, `nltk[everyting]` is installed. This is just a proposal, which is out of the scope of this PR, but clearly is relevant.